wrappers/tensorflow2.2.0_wrapper.py

```diff
@@ -0,0 +1,55 @@
+import pandas
+import tensorflow
+
+import tensorflow_hub
+
+
+class TensorFlowWrapper:
+    """
+    Wrapper object for TensorFlow graph and helps use it.
+    """
+
+    def __init__(self, embedding_layer_hub_name: str) -> None:
+        g = tensorflow.Graph()
+        with g.as_default():
+            # Import the Universal Sentence Encoder's TF Hub module
+            embedding_layer = tensorflow_hub.Module(embedding_layer_hub_name)
+
+            self._sts_input1 = tensorflow.compat.v1.placeholder(tensorflow.string, shape=None)
+            self._sts_input2 = tensorflow.compat.v1.placeholder(tensorflow.string, shape=None)
+
+            # For evaluation we use exactly normalized rather than approximately normalized.
+            sts_encode1 = tensorflow.math.l2_normalize(embedding_layer(self._sts_input1), axis=1)
+            sts_encode2 = tensorflow.math.l2_normalize(embedding_layer(self._sts_input2), axis=1)
+            cosine_similarities = tensorflow.math.reduce_sum(tensorflow.multiply(sts_encode1, sts_encode2),
+                                                        axis=1)
+            clip_cosine_similarities = tensorflow.clip_by_value(cosine_similarities, -1.0, 1.0)
+            self._sim_scores = 1.0 - tensorflow.math.acos(clip_cosine_similarities)
+            init_op = tensorflow.group([tensorflow.compat.v1.global_variables_initializer(), tensorflow.compat.v1.tables_initializer()])
```

Comment:
use tf.disable_v2_behavior() if you'd like to use tf1 behavior or migrate to tensorflow v2.

See previous remark.